.eslintrc.js

```diff
@@ -130,6 +130,7 @@ module.exports = {
       rules: {
         'jest/no-restricted-matchers': 'off',
         'import/unambiguous': 'off',
+        'import/named': 'off',
```

Comment:
Why did this get turned off?

Reply:
Great question! That's because I used `module.exports` in the link below: https://github.com/MetaMask/metamask-extension/pull/10934/files#diff-8fd96322fc664541ebc0326e3c5963e55db825809abb977c6ffad8be7555cbd1R7
After which I started getting these errors:
![image](https://user-images.githubusercontent.com/80175477/116279561-89ff0500-a73c-11eb-9233-62e391f39cce.png)

I will give it little more time to use `export` instead of `module.exports` when re-exporting imported things. Then we hopefully wouldn't need to turn `import/named` off for tests.

Reply:
So, I've changed it to `export` only:
```
export { createSwapsMockStore } from './mock-store';
export { renderWithProvider } from './rendering';
export { setBackgroundConnection } from './background';
export * as MOCKS from './mocks';
export * as CONSTANTS from './constants';
```

But even while tests are green, I'm getting the same ESLint issue when re-exporting:
![image](https://user-images.githubusercontent.com/80175477/116291118-16172980-a749-11eb-90ee-22cad755ec8c.png)

That's why I would prefer to keep `'import/named': 'off',`, just for tests.

Reply:
looks legit. https://github.com/benmosher/eslint-plugin-import/issues/1998

Reply:
Nice find!